server/lib/config/pbench-server-default.cfg

```diff
@@ -73,6 +73,8 @@ rest_uri = /api/v%(rest_version)s
 
 # WSGI gunicorn specific configs
 workers = 3
+# Set the gunicorn worker timeout. Setting it to 0 has the effect of infinite timeouts
+worker_timeout = 0
```

Comment:
Are we sure that we want an _infinite_ time out...as opposed to one which is merely _really big_?

Setting it to 20 minutes, or an hour, or 8 hours, or a day, or something would operate as a safety net while still being "effectively infinite", at least in human terms.

Reply:
Yes definitely. I wasn't sure on this number so just kept it 0 for now, but we can certainly change it to something like 3600 (1 hour) or some other arbitrary number.

Reply:
I recommend setting it to the longest that it would need to be.  That is, I'm pretty sure the client side would give up after 20 minutes (probably long before that), but we might want it to be longer for debugging on the server side...so, perhaps an hour would be too short...but 9000 seconds seems like a reasonable value and nice round number.